src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx

```diff
@@ -15,14 +14,31 @@ import { DisplayField } from './DisplayField';
 export const formatDomains = (domains?: Domain[]): string =>
   (domains ?? []).map((domain: Domain) => domain.domain).join(', ');
 
+const GetAnLEI = (
+  <Link href='https://www.gleif.org/en/about-lei/get-an-lei-find-lei-issuing-organizations'>
```

Comment:
I think this is going to be used elsewhere? Maybe move it over into the pile of other links?